src/App.js

```diff
@@ -24,6 +25,7 @@ function App() {
       <GlobalStyle />
       <Navbar />
       <Route exact path="/" component={LandingPage} />
+      <Route exact path="/schoolssolditems" component={SchoolsSoldItems} />
```

Comment:
This isn't a top level route for App.js...it should be in Dashboard.js.

The correct route path is "/selling/sold" as per the selling navigation Link.